modules/AddBook.js

```diff
@@ -0,0 +1,18 @@
+import bookCollection from './BookCollection.js';
+import displayBooks from './BookManager.js';
+
+function addBook() {
```

Comment:
- [ ] As using ES6 syntax is a requirement in this milestone, kindly consider using ES6 arrow functions instead of random function keyword.

Reply:
Thank you, I have resolved this,